.github/workflows/build-binaries.yml

```
@@ -0,0 +1,113 @@
+            python-version: '3.11'
+          - os: windows-latest
+            platform: win-x64
+            python-version: '3.11'
+
+    runs-on: ${{ matrix.os }}
+    
+    steps:
+    - uses: actions/checkout@v4
+    
+    - name: Set up Python
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)